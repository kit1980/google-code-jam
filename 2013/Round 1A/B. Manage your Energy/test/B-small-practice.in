100
5 1 3
1 2 3
5 1 3
1 3 2
5 1 3
2 1 3
5 1 3
2 3 1
5 1 3
3 1 2
5 1 3
3 2 1
2 1 3
3 1 3
3 1 5
1 2 3 2 1
3 1 5
3 2 1 2 3
5 1 1
2
5 5 10
10 10 10 10 10 10 10 10 10 10
5 1 10
10 1 10 1 10 1 10 1 10 1
5 1 10
1 10 1 10 1 10 1 10 1 10
5 1 10
1 2 3 4 5 6 7 8 9 10
5 1 10
10 9 8 7 6 5 4 3 2 1
5 1 10
1 10 2 9 3 8 4 7 5 6
5 1 10
10 1 9 2 8 3 7 4 6 5
3 3 1
4
4 3 1
1
3 2 1
9
4 2 1
2
5 3 1
3
2 2 1
3
3 3 1
10
5 3 1
7
5 3 1
5
3 1 1
9
3 1 1
5
4 1 1
6
3 3 1
8
4 3 1
1
4 3 1
10
5 4 1
1
2 2 1
3
4 2 1
10
2 1 1
7
3 1 1
1
5 2 1
2
2 1 1
1
5 4 1
7
3 3 1
5
1 1 1
7
2 2 1
8
5 2 1
6
1 1 1
2
1 1 1
3
2 2 1
9
4 1 1
9
1 1 1
7
5 4 1
8
5 3 1
7
4 1 1
7
4 3 1
6
4 4 1
7
1 1 1
10
3 3 1
1
2 1 1
10
4 2 1
1
1 1 1
10
3 1 1
9
4 3 1
8
3 2 1
2
2 2 1
6
4 4 1
1
5 2 1
3
5 2 1
4
5 5 1
1
4 4 1
6
5 1 1
8
3 3 1
3
2 1 1
4
1 1 1
1
5 5 1
5
4 3 1
9
3 3 1
3
1 1 1
4
1 1 1
1
5 2 1
8
1 1 1
1
4 3 1
9
1 1 1
5
4 2 1
4
4 3 1
7
4 1 1
2
1 1 1
2
1 1 1
3
3 2 1
7
4 4 1
7
2 1 1
4
5 5 1
1
5 5 1
10
5 1 2
10 10
5 3 2
2 10
5 2 2
9 10
5 2 3
5 8 10
5 1 4
6 10 5 1
5 1 5
8 8 10 4 9
5 4 6
10 3 10 8 8 3
5 1 7
10 7 5 7 1 10 3
5 4 10
6 9 2 4 4 6 10 9 3 8
