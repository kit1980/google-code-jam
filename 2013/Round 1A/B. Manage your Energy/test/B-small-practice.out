Case #1: 18
Case #2: 18
Case #3: 19
Case #4: 18
Case #5: 19
Case #6: 18
Case #7: 12
Case #8: 15
Case #9: 20
Case #10: 10
Case #11: 500
Case #12: 131
Case #13: 131
Case #14: 95
Case #15: 95
Case #16: 111
Case #17: 115
Case #18: 12
Case #19: 4
Case #20: 27
Case #21: 8
Case #22: 15
Case #23: 6
Case #24: 30
Case #25: 35
Case #26: 25
Case #27: 27
Case #28: 15
Case #29: 24
Case #30: 24
Case #31: 4
Case #32: 40
Case #33: 5
Case #34: 6
Case #35: 40
Case #36: 14
Case #37: 3
Case #38: 10
Case #39: 2
Case #40: 35
Case #41: 15
Case #42: 7
Case #43: 16
Case #44: 30
Case #45: 2
Case #46: 3
Case #47: 18
Case #48: 36
Case #49: 7
Case #50: 40
Case #51: 35
Case #52: 28
Case #53: 24
Case #54: 28
Case #55: 10
Case #56: 3
Case #57: 20
Case #58: 4
Case #59: 10
Case #60: 27
Case #61: 32
Case #62: 6
Case #63: 12
Case #64: 4
Case #65: 15
Case #66: 20
Case #67: 5
Case #68: 24
Case #69: 40
Case #70: 9
Case #71: 8
Case #72: 1
Case #73: 25
Case #74: 36
Case #75: 9
Case #76: 4
Case #77: 1
Case #78: 40
Case #79: 1
Case #80: 36
Case #81: 5
Case #82: 16
Case #83: 28
Case #84: 8
Case #85: 2
Case #86: 3
Case #87: 21
Case #88: 28
Case #89: 8
Case #90: 5
Case #91: 50
Case #92: 60
Case #93: 56
Case #94: 68
Case #95: 76
Case #96: 62
Case #97: 84
Case #98: 185
Case #99: 103
Case #100: 266
